Library/Homebrew/sandbox.rb

```diff
@@ -10,16 +10,16 @@ def self.available?
     OS.mac? && File.executable?(SANDBOX_EXEC)
   end
 
-  def self.formula?(_formula)
+  def self.formula?(_)
     return false unless available?
 
-    !ARGV.no_sandbox?
+    !Homebrew.args.no_sandbox?
```

Comment:
Does this consider `HOMEBREW_NO_SANDBOX`?

Reply:
Intentional change: it wasn't a documented environment variable and I think it's generally a bad idea to have a single env that can disable Homebrew's most potent security measure.